assets/js/src/common/listings/newsletter_status.tsx

```diff
@@ -31,9 +37,12 @@ const NewsletterStatus = ({ scheduledFor, processed, total }: NewsletterStatusPr
     label = `${processed} / ${total}`;
     percentage = 100 * (processed / total);
   } else if (sent) {
-    label = `${total}`;
+    label = `${total} / ${total}`;
```

Comment:
That was a slightly different issue. We said `Sent` which wasn't true in case of failed sending. However, we did _process_ 100% of those emails, which is what the progress indicates. 

I did this change because without it, there would just be a circle with 1 number, which looked a bit confusing ("what this means?"). But when added `X out of Y` it looks more understandable.